### PR TITLE
Add new-driver-only '-explicit-dependency-graph-format=' option

### PR DESCRIPTION
**Pull request by artemcm**

### CODE REVIEW

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1181
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1181
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1181
@swift-ci test Linux platform

File: include/swift/Option/Options.td [hunk 1]
```diff
@@ -1614,10 +1614,15 @@ def use_frontend_parseable_output:
 
 def print_explicit_dependency_graph:
   Flag<["-"], "print-explicit-dependency-graph">,
-  Flags<[HelpHidden, NewDriverOnlyOption]>,
+  Flags<[DoesNotAffectIncrementalBuild, HelpHidden, NewDriverOnlyOption]>,
   HelpText<"Print the result of module dependency scanning after external "
            "module resolution to output">;
 
+def explicit_dependency_graph_format:
```
Comment (bnbarham) [inline include/swift/Option/Options.td:1621]: I imagine this shouldn't be interactive either, but we don't seem very consistent with applying that one/not 😅

**Pull request merged**